lib/rpmsg/rpmsg_virtio.c

```diff
@@ -600,8 +610,15 @@ static void rpmsg_virtio_rx_callback(struct virtqueue *vq)
 
 		metal_mutex_acquire(&rdev->lock);
 		rpmsg_ept_decref(ept);
-		if (rpmsg_virtio_buf_held_dec_test(rp_hdr))
+		if (rpmsg_virtio_buf_held_dec_test(rp_hdr)) {
 			rpmsg_virtio_release_rx_buffer_nolock(rvdev, rp_hdr);
+#if VIRTIO_ENABLED(VQ_RX_EMPTY_NOTIFY)
+			release = true;
```

Comment:
add a  comment  such as:
/* kick will be sent only when last buffer is released */